conveyor/adapter/type.go

```diff
@@ -20,24 +20,28 @@ import (
 	"github.com/insolar/insolar/conveyor/interfaces/iadapter"
 	"github.com/insolar/insolar/conveyor/interfaces/slot"
 	"github.com/insolar/insolar/conveyor/queue"
+	"github.com/insolar/insolar/insolar"
 )
 
 type idType = uint32
 
-// PulseConveyorAdapterTaskSink is iface which helps to slot to push task to adapter
-type PulseConveyorAdapterTaskSink interface {
+// TaskSink is iface which helps to slot to push task to adapter
+// NestedEvent gives access to nested event of adapter
+//go:generate minimock -i github.com/insolar/insolar/conveyor/adapter.TaskSink -o ./ -s _mock.go
+type TaskSink interface {
 	PushTask(respSink AdapterToSlotResponseSink, elementID idType, handlerID idType, taskPayload interface{}) error
-	CancelElementTasks(pulseNumber idType, elementID idType)
-	CancelPulseTasks(pulseNumber uint32)
-	FlushPulseTasks(pulseNumber uint32)
+	CancelElementTasks(pulseNumber insolar.PulseNumber, elementID idType)
+	CancelPulseTasks(pulseNumber insolar.PulseNumber)
+	FlushPulseTasks(pulseNumber insolar.PulseNumber)
 	FlushNodeTasks(nodeID idType)
+	GetAdapterID() uint32
 }
 
 // AdapterToSlotResponseSink is iface which helps to adapter to access to slot
 type AdapterToSlotResponseSink interface {
-	PushResponse(adapterID idType, elementID idType, handlerID idType, respPayload interface{})
-	PushNestedEvent(adapterID idType, parentElementID idType, handlerID idType, eventPayload interface{})
-	GetPulseNumber() uint32
+	PushResponse(adapterID uint32, elementID uint32, handlerID uint32, respPayload interface{})
```

Comment:
Why here `element` param is `uint32`, and in `TaskSink` interface `elementID` param is `idType` ? Shouldn't they be the same type?